api/bolt/migrator/migrate_dbversion23.go

```diff
@@ -1,7 +1,7 @@
 package migrator
 
 import (
-	"github.com/portainer/portainer/api"
+	portainer "github.com/portainer/portainer/api"
```

Comment:
Look at your IDE to fix your auto-import feature, this import update is not required.